cellengine/resources/gate.py

```diff
@@ -15,13 +15,13 @@
 class Gate:
     _id: str = field(on_setattr=readonly)
     experiment_id: str = field(on_setattr=readonly)
-    fcs_file_id: Optional[str]
-    gid: Optional[str]
-    parent_population_id: Optional[str]
-    tailored_per_file: Optional[bool]
     type: str
     x_channel: str
     model: Dict
+    fcs_file_id: Optional[str] = None
+    gid: Optional[str] = None
```

Comment:
(Unrelated, but unless you need this to be optional for some toolkit reason, this shouldn't be optional and should be `field(on_setattr=readonly)`.)